.circleci/config.yml

```diff
@@ -2,27 +2,43 @@ version: 2.1
 
 executors:
   node:
+    parameters:
+        node_version:
+          type: integer
+          default: 10
     docker:
-      - image: circleci/node:10
+      - image: circleci/node:<< parameters.node_version >>
 
 workflows:
   build-test:
     jobs:
-      - prep-deps
+      - prep-deps:
+          name: prep-deps-<< matrix.node_version >>
+          matrix:
+            parameters:
+              node_version: [10, 12, 14]
       - test-lint:
           requires:
-            - prep-deps
+            - prep-deps-10
```

Comment:
Should this also be parameterized?

Reply:
Nope! The intent was to only run the linter on one of the Node versions. 10 being the default choice.

I don't think the linter would behave differently on different node versions :thinking: At least I wouldn't expect it? And even if it does, the consequences are low.

Reply:
That makes sense!